mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -353,8 +353,8 @@ is set to true. Defaults to ZONAL.`,
 						"disk_type": {
 							Type:     schema.TypeString,
 							Optional: true,
-							Default: "PD_SSD",
-							DiffSuppressFunc: caseDiffDashSuppress,
```

Comment:
Question: why are we removing this DSF here? What will API return if users specify `disk_type = "pd-ssd"`?

Reply:
good question. I think we need to keep it